src/admin-views/settings/tickets-emails/message-templates.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @var Array[]  $templates  Array of template info.
```

Comment:
We'll need to document the `@since` in the future, when we implement it. And also have a template title to give more context. Maybe leave a `@todo`